code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -394,6 +394,12 @@
 	var/fortify = 0
 	var/crest_defense = 0
 
+	// Baneling vars
+	/// How long smoke lasts for when baneling explodes
+	var/baneling_smoke_duration = 4
```

Comment:
This is only used in one place and is a xeno var
why not make it a define where its used insstead